src/definitions/modules/slider.js

```diff
@@ -341,10 +320,30 @@ $.fn.slider = function(parameters) {
               module.bind.slidingEvents();
             }
           },
+          touchDown: function(event) {
+            event.preventDefault();  // disable mouse emulation and touch-scrolling
```

Comment:
I wonder, if we probably also need
```
              event.stopImmediatePropagation();
```
because we had this in the previous code for module.bind.touchEvents and it is also used in the mouseevents "mousedown" event.
🤔

Reply:
As far as I know, `stopImmediatePropagation()` prevents following event handlers on the same element and on parent elements from triggering for this event. I don't think skipping other event handlers on the same element is necessary, but it should not hurt either. Skipping possible event handlers on parent elements seems to be a good idea, when the slider is contained in other elements that are listening for touch events. I'll test it and add it if there are no unwanted side-effects.